### ISSUE TITLE
cannot un-WIP or change PR title

### ISSUE THREAD

**Issue by rafipiccolo** [go-gitea/gitea#30030]
### Description

since new version of gitea 1.21.9, i can't un-WIP a PR. i get the error
```
JavaScript error: Cannot read properties of undefined (reading 'endsWith') (https://gitea.raphaelpiccolo.com/assets/js/index.js?v=1.21.9 @ 20:181341). Open browser console to see more details.
```
i also cant change PR title. i get no error, but it does nothing

inspecting the ajax request says :
```
<p>Une erreur s’est produite:</p>
					<pre class="gt-whitespace-pre-wrap gt-break-all">ChangeTitle, exit status 128 - fatal: ambiguous argument &#39;&#39;: unknown revision or path not in the working tree.
Use &#39;--&#39; to separate paths from revisions, like this:
&#39;git &lt;command&gt; [&lt;revision&gt;...] -- [&lt;file&gt;...]&#39;
 - fatal: ambiguous argument &#39;&#39;: unknown revision or path not in the working tree.
Use &#39;--&#39; to separate paths from revisions, like this:
&#39;git &lt;command&gt; [&lt;revision&gt;...] -- [&lt;file&gt;...]&#39;
</pre>
```

### Gitea Version

1.21.9

### Can you reproduce the bug on the Gitea demo site?

Yes

### Screenshots

![image](https://github.com/go-gitea/gitea/assets/4191558/4ab1f46e-ddea-4ab8-a69d-3e05567b8ed2)

### Git Version

2.40.1

### Operating System

ubuntu on the host, i dont know what is in the docker image

### How are you running Gitea?

docker compose
```
gitea:
        image: gitea/gitea:latest
        container_name: gitea
        restart: always
        volumes:
            - ./gitea:/data
```

### Database

MySQL/MariaDB

**Comment by wxiaoguang** [comment]
Here are some workarounds: https://github.com/go-gitea/gitea/issues/29997#issuecomment-2015053135

**Comment by silverwind** [comment]
Duplicate of https://github.com/go-gitea/gitea/issues/29997#issuecomment-2015053135, pinned that one.

**Comment by wxiaoguang** [comment]
Ref #29997

Sorry for the inconvenience.

* It could be fixed by updating to pre-1.21.10:
  * Docker image: use gitea/gitea:1.21-nightly and gitea/gitea:1.21-nightly-rootless
  * Binary: download from https://dl.gitea.com/gitea/1.21/
